app/javascript/components/UsersList.vue

```diff
@@ -32,21 +32,24 @@ export default {
   data: function () {
     return {
       listOfUsers: JSON.parse(this.listUsersAll),
-      theadTitles: ['#', 'First name', 'Last name', 'Email', 'Phone', 'Project' ],
+      theadTitles: ['#', 'First name', 'Last name', 'Email', 'Phone', 'Project'],
```

Comment:
This is good. What you think - is there any ways to improve approach to not create additional array `sortTitles` ?
Please think about it and propose more better approach

Hint: Thanks js for object